query/graphql/src/main/java/org/codice/ddf/admin/graphql/transform/FunctionDataFetcherException.java

```diff
@@ -14,22 +14,23 @@
 package org.codice.ddf.admin.graphql.transform;
 
 import com.google.common.collect.ImmutableMap;
+import java.io.Serializable;
 import java.util.List;
 import java.util.Map;
 import org.boon.Boon;
 import org.codice.ddf.admin.api.report.ErrorMessage;
 
 public class FunctionDataFetcherException extends RuntimeException {
 
-  private List<ErrorMessage> customMessages;
+  private final Serializable customMessages;
 
   public FunctionDataFetcherException(
       String functionName, List<Object> args, List<ErrorMessage> customMessages) {
     super(Boon.toPrettyJson(toMap(functionName, args, customMessages)));
-    this.customMessages = customMessages;
+    this.customMessages = (Serializable) customMessages;
```

Comment:
I still think we should be be adding the `customMessages` to a private serializable list instead. The return type `Serializable` for `getCustomMessages` doesn't make much sense.